docs/globals.html

```diff
@@ -2799,8 +2799,10 @@ <h1>@auth0/auth0-spa-js</h1>
 			</a>
 			<p>Auth0 SDK for Single Page Applications using <a href="https://auth0.com/docs/api-auth/tutorials/authorization-code-grant-pkce">Authorization Code Grant Flow with PKCE</a>.</p>
 			<p><a href="https://circleci.com/gh/auth0/auth0-spa-js"><img src="https://circleci.com/gh/auth0/auth0-spa-js.svg?style=svg" alt="CircleCI"></a>
-				<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/:license-mit-blue.svg?style=flat" alt="License"></a>
-			<a href="https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fauth0-spa-js?ref=badge_shield"><img src="https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fauth0-spa-js.svg?type=shield" alt="FOSSA Status"></a></p>
+				<img src="https://img.shields.io/github/v/release/auth0/auth0-spa-js" alt="Release">
+				<a href="https://codecov.io/gh/auth0/auth0-spa-js"><img src="https://img.shields.io/codecov/c/github/auth0/auth0-spa-js" alt="Codecov"></a>
+				<img src="https://img.shields.io/npm/dw/@auth0/auth0-spa-js" alt="Downloads">
+			<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/:license-mit-blue.svg?style=flat" alt="License"></a></p>
```

Comment:
Is this change intended?

Reply:
Yeah I think this is just rebuilding the readme for docs based on #620